hackathon_site/dashboard/frontend/src/components/teamDetail/TeamInfoTable/TeamInfoTable.tsx

```diff
@@ -10,44 +11,56 @@ import {
     TableRow,
 } from "@material-ui/core";
 import Typography from "@material-ui/core/Typography";
-import { mockTeamMultiple } from "testing/mockData";
 import React from "react";
+import { useSelector } from "react-redux";
+import {
+    isLoadingSelector,
+    teamDetailAdapterSelector,
+} from "slices/event/teamDetailSlice";
 
 export const TeamInfoTable = () => {
+    const isTeamInfoLoading = useSelector(isLoadingSelector);
+    const teamInfo = useSelector(teamDetailAdapterSelector.selectAll);
```

Comment:
Nit pick: a better name for this const would be "profiles"

Reply:
k changed